server.js

```diff
@@ -24,7 +14,52 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send(
+    'Golden Globe! /shows?country="Put in country" - /shows?title="Put in Title"'
+  )
+})
+
+app.get('/listlength', (req, res) => {
+  res.json(goldenGlobesData.length)
+})
```

Comment:
I guess you just wanted to test this, but this is not really a super useful endpoint. and not so RESTful